pkg/mountutil/mountutil_linux.go

```diff
@@ -103,13 +103,16 @@ func parseVolumeOptionsWithMountInfo(vType, src, optsRaw string, getMountInfoFun
 	var (
 		writeModeRawOpts   []string
 		propagationRawOpts []string
+		bindOpts           []string
 	)
 	for _, opt := range strings.Split(optsRaw, ",") {
 		switch opt {
 		case "rw", "ro", "rro":
 			writeModeRawOpts = append(writeModeRawOpts, opt)
 		case "private", "rprivate", "shared", "rshared", "slave", "rslave":
 			propagationRawOpts = append(propagationRawOpts, opt)
+		case "bind", "rbind":
+			bindOpts = append(bindOpts, opt)
```

Comment:
docker volume do not support `bind` and `rbind` mode

Reply:
https://github.com/containerd/nerdctl/compare/9354b2903ec210dbc6a7d3c021f708917fcb7432..fcc8cf9ac2ea8583d7390e94fcb70a50cd7b7ac3

Reply:
looks good now